index.d.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export type HTMLAttributes = Record<string, string | number | boolean | readonly string[]>;
```

Comment:
This `HTMLAttributes` identifier was raising a lint error because it is not StrictPascalCase under [`@typescript-eslint/naming-convention`](https://typescript-eslint.io/rules/naming-convention/). If it were me, I would change the casing, but this would be an arguably unnecessary breaking change + you may just prefer it as is.